QModManager/Patching/QModPlaceholder.cs

```diff
@@ -36,7 +36,7 @@ internal QModPlaceholder(string name, ModStatus status)
             this.Author = "Unknown";
             this.SupportedGame = QModGame.None;
             this.Enable = false;
-            this.Status = ModStatus.UnidentifiedMod;
+            this.Status = status;
```

Comment:
Do the opposite.  
Remove the parameter and set the status to `ModStatus.UnidentifiedMod` for the `QModPlaceholder` .

Reply:
There are a lot of calls to `QModPlaceholder`, and each of them have a different `ModStatus`. I don't see why I should remove that.

Reply:
There are? I guess I'll have to look these over again.  
For now, your fix is fine.